lib/jsdom/browser/Window.js

```diff
@@ -361,6 +363,14 @@ function Window(options) {
     window: this
   }).interface;
 
+  const AbortSignalWrapper = createAbortSignal({
```

Comment:
Sadly we require window because EventTarget uses it to make sure we're attached to a document/not closed. We should fix that some day (same problem in XHR, I think), but ... yak-shaving.